### PR TITLE
Jest test

### PR DESCRIPTION
**Pull request by Kel-nana**
### Jest Test

- Use jest framework for [`testing`](https://github.com/Kel-nana/Review-To-do-list/blob/1028bf67985071a841137ed7ff7c75bf662ada71/package.json#L7).
- Create a test file [`([..].test.js)`](https://github.com/Kel-nana/Review-To-do-list/blob/peer-review/src/tests/todo.test.js) for a file containing the add item and delete item functions that you want to test.
- Mocked function [`setItem`](https://github.com/Kel-nana/Review-To-do-list/blob/1028bf67985071a841137ed7ff7c75bf662ada71/src/tests/todo.js#L1)
- Test if [`add`](https://github.com/Kel-nana/Review-To-do-list/blob/1028bf67985071a841137ed7ff7c75bf662ada71/src/tests/todo.test.js#L4-L20) and [`remove`](https://github.com/Kel-nana/Review-To-do-list/blob/1028bf67985071a841137ed7ff7c75bf662ada71/src/tests/todo.test.js#L22-L38) functions  work
- Make sure you group your tests using the [`describe()`](https://github.com/Kel-nana/Review-To-do-list/blob/1028bf67985071a841137ed7ff7c75bf662ada71/src/tests/todo.test.js#L4) method

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,755 @@
...
+    "@babel/plugin-transform-modules-commonjs": "^7.21.2",
+    "babel-eslint": "^10.1.0",
+    "babel-jest": "^29.5.0",
+    "css-loader": "^6.7.3",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "hint": "^2.0.0",
+    "html-webpack-plugin": "^5.5.0",
+    "jest-environment-jsdom": "^29.5.0",
```
Comment (rotimiazeez) [inline package.json]: - Please ensure `Jest` is present in your `devDependencies` so your code can be tested 👍🏽
Screenshot from 2023-03-15 19-29-52
- You can add it by pasting the below dependencies into your devDependencies
```
"jest": "^29.4.3",
"jest-localstorage-mock": "^2.4.26",
```

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Kel-nana 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: package.json (hunk 1 above)
Comment (Kel-nana) [reply package.json]: Addition of jest in [`dependencies`](https://github.com/Kel-nana/Review-To-do-list/blob/8f0fded52eaa5b5f4d5616b95be5dec0809d6807/package.json#L29-L31)

**Review by Shedrack-Sunday — APPROVED** [review]
# STATUS: APPROVED ♻️♻️
Hi @Kel-nana 

To highlight 👍🏽
✔️ Navigations work as expected
✔️ No linter errors

Your project is complete! There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉

## Confirmed test:
image


## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**